### PR TITLE
VDS-91: Fix sonarcloud issues

### PR DESCRIPTION
**Pull request by pushnitsa**
Description: 

--

QA-test: VDS-1707

Demo-test: VDS-1708

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-demo-theme-default-1.9.0-pr-23-d481e08d.zip

### CODE REVIEW

File: assets/js/account/account-quotes.tpl.liquid [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <div class="one-half">
     <h4>{{ 'customer.quotes.title' | t }}</h4>
-    <table class="full" ng-if="$ctrl.entries.length">
+    <table class="full" ng-if="$ctrl.entries.length" aria-label=" ">
```
Comment (trueboroda) [inline assets/js/account/account-quotes.tpl.liquid]: aira-label should have an accessible name. Normal meaningful name.

File: assets/js/account/account-quotes.tpl.liquid [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
         <thead>
             <tr>
-                <th>{{ 'customer.quotes.quote_number' | t }}</th>
-                <th>{{ 'customer.quotes.date' | t }}</th>
-                <th>{{ 'customer.quotes.status' | t }}</th>
+                <th id="quote_number">{{ 'customer.quotes.quote_number' | t }}</th>
```
Comment (trueboroda) [inline assets/js/account/account-quotes.tpl.liquid]: It is recommended to add scope attributes to <th> headers whenever possible.
<th scope="col">{{ 'customer.quotes.quote_number' | t }}</th>
<th scope="col">{{ 'customer.quotes.date' | t }}</th>
<th scope="col">{{ 'customer.quotes.status' | t }}</th>

File: assets/js/lists/account-lists.tpl.liquid [hunk 3]
```diff
@@ -46,7 +46,7 @@
             </span>
             <div  class="ajax-informer text-center" ng-class="{ 'fade': $ctrl.accountLists.productAdded }">{{ 'wishlist.general.added_to_cart' | t}}</div>
 
-            <table class="vc-product-table list-table">
+            <table class="vc-product-table list-table" aria-hidden="true">
```
Comment (trueboroda) [inline assets/js/lists/account-lists.tpl.liquid:49]: The more preferable way is to use role="presentation".

No issue will be raised on <table> used for layout purpose, i.e. when it contains a role attribute set to "presentation" or "none".

File: templates/customers/order.liquid [hunk 4]
```diff
@@ -37,14 +37,14 @@
...
-    <table class="full">
+    <table class="full" aria-hidden="true">
       <thead>
         <tr>
-          <th>{{ 'customer.order.product' | t }}</th>
-          <th>{{ 'customer.order.sku' | t }}</th>
-          <th>{{ 'customer.order.price' | t }}</th>
-          <th class="text-center">{{ 'customer.order.quantity' | t }}</th>
-          <th class="text-right">{{ 'customer.order.total' | t }}</th>
+          <th id="order_product">{{ 'customer.order.product' | t }}</th>
```
Comment (trueboroda) [inline templates/customers/order.liquid]: Use scope="col" for th

File: templates/customers/order.liquid [hunk 5]
```diff
@@ -37,14 +37,14 @@
     </div>
     {% endif %}
 
-    <table class="full">
+    <table class="full" aria-hidden="true">
```
Comment (trueboroda) [inline templates/customers/order.liquid]: Use aria-label instead of aria-hidden

File: templates/customers/quote-requests.liquid [hunk 6]
```diff
@@ -6,12 +6,12 @@
         <h4>{{ 'customer.quotes.title' | t }}</h4>
         {{ paginator = customer.quote_requests | paginate 5 }}
         {% if customer.quote_requests.size > 0 %}
-        <table class="full">
+        <table class="full" aria-hidden="true">
```
Comment (trueboroda) [inline templates/customers/quote-requests.liquid]: use aria-label

File: templates/customers/quote-requests.liquid [hunk 7]
```diff
@@ -6,12 +6,12 @@
...
             <thead>
                 <tr>
-                    <th>{{ 'customer.quotes.quote_number' | t }}</th>
-                    <th>{{ 'customer.quotes.date' | t }}</th>
-                    <th>{{ 'customer.quotes.status' | t }}</th>
+                    <th id="qoute_number">{{ 'customer.quotes.quote_number' | t }}</th>
```
Comment (trueboroda) [inline templates/customers/quote-requests.liquid]: Use scope="col"

File: templates/quote-request.liquid [hunk 8]
```diff
@@ -41,7 +41,7 @@
         </div>
         <div class="grid" ng-if="displayForStatuses(['Processing', 'Proposal sent', 'Ordered', 'Rejected', 'Canceled'])">
             <div class="grid-item medium-down--one-whole">
-                <table>
+                <table aria-hidden="true">
```
Comment (trueboroda) [inline templates/quote-request.liquid]: Use role="presentation"

File: templates/thanks.liquid [hunk 9]
```diff
@@ -1,13 +1,13 @@
 <div ng-controller="orderController">
     <h4>{{ 'customer.order.title' | t: order.number }}</h4>
-    <table class="full">
+    <table class="full" aria-hidden="true">
```
Comment (trueboroda) [inline templates/thanks.liquid]: Use area-label

File: templates/thanks.liquid [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
         <thead>
             <tr>
-                <th>{{ 'customer.order.product' | t }}</th>
-                <th>{{ 'customer.order.sku' | t }}</th>
-                <th>{{ 'customer.order.price' | t }}</th>
-                <th class="text-center">{{ 'customer.order.quantity' | t }}</th>
-                <th class="text-right">{{ 'customer.order.total' | t }}</th>
+                <th id="order_product">{{ 'customer.order.product' | t }}</th>
```
Comment (trueboroda) [inline templates/thanks.liquid]: Use scope="col"

**Review by trueboroda — CHANGES_REQUESTED** [review]

File: assets/js/lists/account-lists.tpl.liquid (hunk 3 above)
Comment (pushnitsa) [reply assets/js/lists/account-lists.tpl.liquid:49]: In this case, sonar produces another issue :)

File: templates/quote-request.liquid [hunk 11]
```diff
@@ -41,7 +41,7 @@
         </div>
         <div class="grid" ng-if="displayForStatuses(['Processing', 'Proposal sent', 'Ordered', 'Rejected', 'Canceled'])">
             <div class="grid-item medium-down--one-whole">
-                <table>
+                <table aria-label="Quote request">
```
Comment (trueboroda) [inline templates/quote-request.liquid:44]: Tabel serves for layout purposes. Maybe it is better to use aria-hidden="true" or set  role="presentation" and mark this sonar rule as "will not be fixed".

**Review by trueboroda — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**